.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+# This GitHub action can publish assets for release when a tag is created.
+# Currently its setup to run on any tag that matches the pattern "v*" (ie. v0.1.0).
+#
+# This uses an action (paultyng/ghaction-import-gpg) that assumes you set your
+# private key in the `GPG_PRIVATE_KEY` secret and passphrase in the `PASSPHRASE`
+# secret. If you would rather own your own GPG handling, please fork this action
+# or use an alternative one for key handling.
+#
+# You will need to pass the `--batch` flag to `gpg` in your signing step
+# in `goreleaser` to indicate this is being used in a non-interactive mode.
+#
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  test:
```

Comment:
The `test` job should still run on all `pull_request` and `push` events as before: https://github.com/claranet/terraform-provider-zabbix/pull/19/files#diff-c2e34dca95d7bdcdd2e85f92915cdda8L3-L5

Then the `if` condition from the old workflow should be added to the `release`: https://github.com/claranet/terraform-provider-zabbix/pull/19/files#diff-c2e34dca95d7bdcdd2e85f92915cdda8L25

That's why the workflow was named `Test and Release`.

Or maybe we could separate the test job from the release job in another workflow.

WDYT?

Reply:
fixed